src/index.ts

```diff
@@ -48,6 +48,8 @@ export const GSN_MUMBAI_PAYMASTER_CONTRACT_ADDRESS =
   '0xA99D942723748EB28f1AC25434F6bB6184C29018'
 export const KETL_OBSS_CONTRACT_ADDRESS =
```

Comment:
Maybe it's better to call it `DEV_KETL_OBSS_CONTRACT_ADDRESS`? Like you did with `PROD_KETL_OBSS_CONTRACT_ADDRESS`